target/linux/pistachio/base-files/etc/config/wireless

```diff
@@ -0,0 +1,6 @@
+config wifi-device  radio0
```

Comment:
This file should be generated at first boot by the mac80211.sh script when this devices gets detected.